buildscripts/scripts/mpv.sh

```diff
@@ -2,23 +2,28 @@
 
 . ../../include/path.sh
 
+build=_build$ndk_suffix
+
+
+# Meson must clean the build directory or it will build a static library instead of a shared one
+rm -rf $build
+
 if [ "$1" == "build" ]; then
 	true
 elif [ "$1" == "clean" ]; then
-	rm -rf _build$ndk_suffix
 	exit 0
 else
 	exit 255
 fi
 
-[ -f waf ] || ./bootstrap.py
+unset CC CXX # meson wants these unset
 
-PKG_CONFIG="pkg-config --static" \
-./waf configure \
-	--disable-iconv --lua=52 \
-	--enable-libmpv-shared \
-	--disable-manpage-build \
-	-o "`pwd`/_build$ndk_suffix"
+meson $build --cross-file "$prefix_dir"/crossfile.txt \
+	--default-library=shared \
```

Comment:
(meson documentation uses this style)
```suggestion
	--default-library shared \
```